src/main/resources/plugin.yml

```
@@ -40,3 +40,7 @@ permissions:
 
   ouroboros.mines.command.dropgroup:
     default: op
+
+  ouroboros.mines.autopickup:
+    description: Grants permission to make use of the autopickup feature - if enabled.
+    default: op
```

`op` as default could lead to the problem I had mentioned before.
We could keep the `or` in line 123 of BlockBreakListener and change `default` to `false`.

```suggestion
    default: false
```